pynest/nest/tests/test_sp/test_enable_multithread.py

```diff
@@ -29,7 +29,10 @@
 # and multiple threads are set, or if multiple threads are set and
 # the enable_structural_plasticity function is called.
 
+HAVE_OPENMP = nest.sli_func("statusdict/threading ::")
```

Comment:
It would be even better to do as follows:
```Python
HAVE_OPENMP = nest.sli_func("is_threaded")
```
This insulates the details of how threading is marked at the sli level. The result is a bool.